build.gradle

```diff
@@ -22,9 +22,12 @@ dependencies {
     api ('com.mesosphere.usi:core') { version { branch = usiBranch } }
     api ('com.mesosphere.usi:core-models') { version { branch = usiBranch } }
     api ('com.mesosphere.usi:mesos-client') { version { branch = usiBranch } }
+    api 'com.squareup.okhttp3:okhttp:3.14.1'
```

Comment:
If someone want to replace the `OkHttpClient` with Akka HTTP feel free to do so. I'm out for the next week 🙂

Reply:
nah premature optimization; I'm sure a blocking client is just fine for Jenkins

Reply:
oh and it's test code LOL so NO

Reply:
yup agreed this is fine for testing,